content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts.md

```diff
@@ -52,10 +52,15 @@ Alternatively, if you want to use the HTTPS protocol for both accounts, you can
 {% data reusables.git.confirm-credential-manager %}
 {% data reusables.git.clear-the-stored-credentials %}
    {% data reusables.git.no-credential-manager %}
-   * If the output is `osxkeychain`, you're using the macOS keychain. To clear the credentials, enter the following command.
+   * If the output is `osxkeychain`, you're using the macOS keychain. To clear the credentials, enter the following command and interactive inputs:
 
      ```shell copy
```

Comment:
also when mixing prompt and output, the whole example should not be copied perhaps?
```suggestion
     ```shell
```